src/core.c/REPL.rakumod

```diff
@@ -315,6 +331,10 @@ do {
         }
 
         method init(Mu \compiler, $multi-line-enabled --> Nil) {
+            if not compiler.is-interactive || $*IN.t {
```

Comment:
Ow man, the 'not' is sneaky without parens.

Reply:
Wait no, was this meant to be `compiler.is-interactive && !$*IN.t`?

Reply:
No, I was avoiding parens. I find 'if not' to be less "sneaky" than 'unless' when paired with other conditional operators.

But since there is controversy, I will change it.

Reply:
I think it's much clearer in the latest iteration too:

```
if compiler.repl-mode eq 'tty' && not $*IN.t {
```